README.md

```diff
@@ -1,2 +1,10 @@
-# helm-charts
-Helm charts for StackRox Kubernetes Security Platform
+# Helm charts for the StackRox Kubernetes Security Platform
+
+To use Helm charts to install the [StackRox Kubernetes Security Platform](https://www.stackrox.com/platform/)
+follow the instructions in the `README.md` file under the corresponding version
```

Comment:
StackRox version

Reply:
An example would be nice. If you are deploying 3.0.41.4 use the README at <link here> ? @gaurav-nelson thoughts?

Reply:
Thanks @boo, I have included this in my suggestion.